cmd/node/config/gasSchedules/gasScheduleV1.toml

```diff
@@ -184,6 +184,7 @@
     UnmarshalECC           = 20000
     UnmarshalCompressedECC = 270000
     GenerateKeyECC         = 7000000
+    EncodeDERSig           = 10000000
```

Comment:
what does DER stands for?

Reply:
The Distinguished Encoding Rules (DER) format is used to encode ECDSA signatures in Bitcoin